osquery/tables/system/windows/logged_in_users.cpp

```diff
@@ -119,6 +119,26 @@ QueryData genLoggedInUsers(QueryContext& context) {
       r["host"] = std::string(addr, CLIENTADDRESS_LENGTH);
     }
 
+    if (!r["host"].length()) {
```

Comment:
I think we should be changing this just to an `else` and connect it to the above if/else if chain, since `r["host"]` can be empty really only because it's not one of the type of addresses above (in fact it's AF_UNSPEC).

Reply:
Okk, done